### PR TITLE
Meta: Use shiny new hostname

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -43,15 +43,15 @@
   }
 </style>
 <script>
```
Comment (annevk) [inline spec.html]: This entire script block is obsolete I think. I suggest removing it as per #1575 and then I can close that.

Comment (mathiasbynens) [reply spec.html]: Done. (Also, sorry, I should have seen your PR earlier.)

**Comment by michaelficarra** [comment]
So... what's with the Spanish TLD?

**Comment by ljharb** [comment]
I’m a bit confused by the new domain name as well; was it discussed during the meeting?

**Comment by annevk** [comment]
Yes and it had consensus.

**Comment by ljharb** [comment]
It’s not on the agenda; maybe i missed it in the draft notes. I’ll take another look.

**Comment by michaelficarra** [comment]
@ljharb It's in the June 4th notes. Not much is mentioned on motivation though. Only comments are from @gesa and @ecmageneva. @ecmageneva requested a Swiss registrar and, Ecma being a swiss organisation, I would think that tc39.ch would be a more appropriate ccTLD. Should we move this discussion to the reflector?

**Comment by ljharb** [comment]
Thanks. Yeah a reflector issue would be best.

**Comment by mathiasbynens** [comment]
Meanwhile, the domain is live and the old domain redirects to it. Let's merge this PR to reflect reality (and save anyone who clicks a redirect)?

**Comment by mathiasbynens** [comment]
@ljharb or @zenparsing, could you also update the URL here?

**Comment by ljharb** [comment]
Done.

**Comment by azu** [comment]
FYI: https://www.ecma-international.org/publications/standards/Ecma-262.htm should also be updated when the new domain has been decided.

**Comment by michaelficarra** [comment]
@ljharb posted to discourse: https://es.discourse.group/t/draft-spec-text-domain/75

**Pull request merged**